druid-shell/Cargo.toml

```diff
@@ -16,19 +16,20 @@ use_gtk = ["gtk", "gtk-sys", "gio", "gdk", "gdk-sys", "glib", "glib-sys", "cairo
 default-target = "x86_64-pc-windows-msvc"
 
 [dependencies]
-piet-common = "0.0.9"
+piet-common = "0.0.10"
 log = "0.4.8"
 lazy_static = "1.0"
 time = "0.2.7"
 cfg-if = "0.1.10"
+# NOTE: if changing, ensure version is compatible with the version in piet
 kurbo = "0.5.11"
```

Comment:
So we're keeping the direct kurbo dependency and not going back to piet's re-export? I guess it can be useful again if we expect fast changes in kurbo.

Reply:
That was my thinking, yea.